### ISSUE TITLE
Wrong version number info outputted by latest release

### ISSUE THREAD

**Issue by aucompbiker** [atc0005/go-lockss#203]
Adam,

In the latest release (v0.2.0) the executable is outputting "go-lockss dev build" for the version number instead of the project name's actual version number info (i.e. v0.2.0).

Example output from running **hayoh** executable on lockss-adpn2.  The executable was installed using the go-lockss-v0.2.0.x86_64.rpm package.  

```
[zrbdmas@lockss-adpn2 ~]$ hayoh --version

go-lockss dev build
https://github.com/atc0005/go-lockss
```
The output is the same when running the uncompressed binary **hayoh-linux-amd64**.
```
[zrbdmas@lockss-adpn2 ~]$ ./hayoh-linux-amd64 --version

go-lockss dev build
https://github.com/atc0005/go-lockss
```

**Comment by atc0005** [comment]
@aucompbiker Thanks for reporting this. Will get this fixed in the next release.

**Comment by atc0005** [comment]
@aucompbiker

This should be resolved in the v0.2.1 release:

- https://github.com/atc0005/go-lockss/releases/tag/v0.2.1

Thanks again for reporting this.
